packages/ui-tree-browser/src/TreeBrowser/README.md

```diff
@@ -137,358 +151,14 @@ class Example extends React.Component {
         onCollectionToggle={this.handleCollectionToggle}
         onCollectionClick={this.handleCollectionClick}
       />
+      </ApplyTheme>
     )
   }
 }
 
 render(<Example/>)
 ```
 
-### Customizing Icons
-
-All of the `<TreeBrowser>` icons are customizable. The following example sets custom icons for the expanded and collapsed state of the collections along with each of the items.
-
```

Comment:
Why did you delete so much from its Readme? These look like useful examples.

Reply:
Oh, my mistake, was using the Readme for testing and forgot to revert before the commit. Fixed!